front/src/applications/editor/tools/pointEdition/components.tsx

```diff
@@ -212,6 +212,15 @@ export const PointEditionLeftPanel: FC<{ type: EditoastType }> = <Entity extends
     | { type: 'ready'; id: string; track: TrackSectionEntity }
   >({ type: 'idle' });
 
+  // Hack to be able to launch the submit event from the rjsf form by using
+  // the toolbar button instead of the form one.
```

Comment:
You can even add a link to the github issue
```suggestion
  // Hack to be able to launch the submit event from the rjsf form by using
  // the toolbar button instead of the form one.
  // See https://github.com/rjsf-team/react-jsonschema-form/issues/500
```

Reply:
Yes good idea